features/test/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import typescript from "@rollup/plugin-typescript";
+import copy from "rollup-plugin-copy";
+
+export default {
+    input: "src/test.ts",
+    output: {
+        dir: "dist",
+        format: "cjs",
+    },
```

Comment:
The faucet config said `exports: "testFeature"` -> Not entirely sure what it does, but maybe it defines a global variable this is assigned to? We don't get the same behaviour out of the box from Rollup, do we?

Reply:
I think it would be equivalent to "globals". I would need to check it more extensively. I've just loaded it up in Chromium and it does not seem to yield any error... But of course more extensive testing would be needed. OTOH, it could simply be dead code.

Reply:
Um, it's not. Let me fix that.

Reply:
Fixed with be6a6c6